template/razor.go

```diff
@@ -26,7 +27,7 @@ func (t *Template) applyRazor(content []byte) (result []byte, changed bool) {
 		}
 	}
 	content = []byte(strings.Replace(string(content), funcCall, "", -1))
-	InternalLog.Debugf("Generated content\n\n%s\n", color.HiCyanString(String(content).AddLineNumber(0).Str()))
+	InternalLog.Infof("Generated content\n\n%s\n", color.HiCyanString(String(content).AddLineNumber(0).Str()))
```

Comment:
Info? Seems more like debug information to me. Consider a file that generates thousands of lines. That would crush your console

Reply:
I just restored the logging level to what they were before your first reclassification. According that notice no longer existed.

notice -> Info
Info -> debug
debug -> trace

notice was higher priority than info, but after your rework, all notices became lower than info.